Terminal.Gui/Drawing/Region.cs

```diff
@@ -1,4 +1,8 @@
-﻿/// <summary>
+﻿#nullable enable
```

Comment:
FWIW, in https://github.com/gui-cs/Terminal.Gui/pull/3886, i've rewritten Region, so this is is all moot. I'd love your help on that PR once I'm ready tho!